### ISSUE TITLE
[Help Wanted] Arch + Hypuccin

### ISSUE THREAD

**Issue by 0xSDX** [MathisP75/hyppuccin#1]
I recently came across this rice and it has since been on my mind. 
I have a similar family laptop (Legion Y540(i5/1650)). 
I intend to achieve :
- Replicating this rice with ArchLinux without actually installing a DE
- Proper Dual GPU Hardware drivers configured (iGPU+dGPU)
Do let me know, where I can contact you for the help and doubts. 

To : @MathisP75 thankyou for this awesome 🍚 showcase and configs.

**Comment by MathisP75** [comment]
Hi! I can surely help you with that, as I achieved it myself with very similar hardware. I have mine setup so it's usable as a laptop and when plugged into an external monitor. Unfortunately, there are problems when using both GPUs at the same time, so in laptop mode I am limited to my intel GPU and I use my Nvidia one when plugged on an external monitor, which doesn't allow me to use the laptop monitor at the same time. Despite this, it's a very usable setup and I would gladly guide to to replicate it. I think most of my configs should work out of the box for you. I will follow with another reply later to explain how it works exactly.

**Comment by MathisP75** [comment]
Ok, so first of all, you will want to follow the steps on the hyprland wiki to set it up with your Nvidia card. Once that is done and you are able to run hyprland, you will need a wrapped launcher, which is also explained on the wiki. I added mine in the repo. It checks if there are external monitor connected and forces  the Nvidia card to be used if it is the case, else it doesn't. You might have to make some changes if the name of your outputs (DP-1, DP-2...) don't match. The wrapped launcher also exports some important environment variables. I added one to  indicate whether it is in laptop or desktop mode so that I can launch a modified waybar and wallpaper instance. That's all I can think of for now, but don't hesitate to join the hyprland Discord server if you haven't already, where I and many others can help you.
